overview/index.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: index
+title: Overview
+permalink: /overview/
+---
+
+This page provides an overview of how Kroxylicious works.  For more details, please refer to the [documentation](./kroxylicious).
+
+#### What is Kroxylicious?
+
+Kroxylicious is an Apache Kafka&#174; protocol-aware proxy.  It can be used to layer uniform behaviours onto a
+Kafka based system in areas such as data-governance, security, policy enforcement and audit without needing to
+change either the applications or the Kafka Cluster.
+
```

Comment:
Need a section here explaining that Kroxylicious is deployed as a component separate, between the applications and the brokers.